### PR TITLE
24 sexgender mapping is incorrect

### PR DESCRIPTION
**Pull request by PeterKranich**
## 📑 Description

Reworked PID-8 sex mapping.

## ☑ Mandatory Tasks

The following aspects have been respected by the pull request assignee and at least one reviewer:
  
- Changelog update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer

### CODE REVIEW

File: asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc [hunk 1]
```diff
@@ -206,43 +206,42 @@ xsd:date: *2001-10-26* -> HL7 DTM: *20011026*
...
+====
+NOTE: <<ref_tbl_pid8_mapping>> defines the mapping of the SDC patient's sex information to the data fields of the HL7 data type *IS* used in the PID-8 field.
+====
+****
 
 .R8107
 [sdpi_requirement#r8107,sdpi_req_level=shall,sdpi_max_occurrence=1]
 ****
-If <<r8102>> is met, then a <<actor_somds_dec_gateway>> / <<actor_somds_acm_gateway>> shall set the PID-8 field to the code for the administrative sex.
+If <<r8102>> is met and the <<actor_somds_v2_gateway>> is configured to export the patient's sex in the PID-8 field, then a <<actor_somds_dec_gateway>> / <<actor_somds_acm_gateway>> shall set the PID-8 field to the code for the administrative sex.
```
Comment (riechkathrin) [inline asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc:242]: Rephrase to something like "The <actor> MAY provide the HDO with means to configure the <actor> to export the patient’s sex in the PID-8 field.”

File: asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc [hunk 2]
```diff
@@ -206,43 +206,42 @@ xsd:date: *2001-10-26* -> HL7 DTM: *20011026*
...
 
-However, in the clinical context of a point of care device the biological sex, or sex at birth is important for various algorithms.
+The organization has to consider the risk that the patient's sex set in the <<acronym_mdib>> and mapped to the PID-8 field does not lead to a misinterpretation of the sex concept on <<actor_somds_v2_gateway>> consumer side.
```
Comment (riechkathrin) [inline asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc:244]: Should be a shall requirement:
If the <actor> provides the HDO with means to configure the <actor> to export the patient’s sex in the PID-8 field, the <actor> SHALL require in the ACCOMPANYING INFORMATION that the HDO considers the risk of misinterpretation.

File: asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc (hunk 1 above)
Comment (PeterKranich) [reply asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc]: Reworded R8107

File: asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc (hunk 2 above)
Comment (PeterKranich) [reply asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc]: Added requirement R8122

**Review by riechkathrin — APPROVED** [review]

**Pull request merged**